addons/dark-www/experimental_scratchwww.css

```diff
@@ -62,7 +62,8 @@ body:not(.sa-body-editor) .join-flow-title,
 .download .installation,
 body:not(.sa-body-editor) .gender-radio-row,
 .wedo .banner,
-.studio-report-modal .studio-report-tile-header {
+.studio-report-modal .studio-report-tile-header,
+.studio-info-box.studio-info-box-success {
```

Comment:
I don't think green boxes should be made gray.

dark-www currently has little support for various messages and errors, so it would probably be better to fix all of them at once.

Reply:
Should all dark-www changes be removed from this PR?

Reply:
and scratchr2 changes?

Reply:
Those specific to the promote modal itself can stay.

Reply:
@mxmou The color is not Scratch dark green (or "extension green"/"ui-aqua"); it's more lighter than that. IMO we should change it to a darker color.

I'll remove that line from this PR but the color definitely needs some changes.